resources/similarity.py

```diff
@@ -0,0 +1,10 @@
+from resources.embedding import inception_v3
+
+### import other similarity measure methods
+
+
+def get_similarities(img, args):
```

Comment:
```suggestion
def get_similarities_vectors(img, args):
```
Maybe this name is clearer?

At the end, the function should return two lists: one containing the embedding vector and the other containing the color vector. Both lists should be inserted into a pickle file along with the corresponding image ID at the same time: return [[embedding_vector], [color_vector]]. What do you think about it?